src/main/resources/sample_config.properties

```diff
@@ -0,0 +1,29 @@
+! A sample configuration file for FBase
+! You should create a local.properties based on this
+
+!!! Machine specific data (each machine of the node can have different values here) !!!
+machineName: <the machine name (MANDATORY)>
```

Comment:
do we need that? we already have IP+port for those.

Reply:
The ip address might change if it is dynamic. I likes the idea of being able to identify a machine by a name that is not changed as much (or not at all).